rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -943,13 +940,10 @@ private final class BackoffRefreshListener implements ChildLbStatusListener {
 
     @Override
     public void onStatusChanged(ConnectivityState newState) {
-      logger.log(ChannelLogLevel.DEBUG, "LB status changed to: {0}", newState);
```

Comment:
This one seems like it could be useful.

Reply:
Nope! Because it has no context! And the way this listener is arranged, _it_ doesn't know which child this came from. Knowing the 'target' for the child would be necessary to make it helpful.

It is called every single picker update, so it is noisy at best. If there's only one child, it isn't useful because RLS will pass-through its state and the channel will log it.

It was too broken for me to try to fix, and it is too noisy. Thus delete. I'd be fine if later someone adds something back in that's better.

I did leave the "Transitioning from TRANSIENT_FAILURE to READY" because it was a tell that we were refreshing backoffs. But I _just_ realized that that transition check is broken _because it doesn't have context_! It is comparing updates across child policies! And I had already wondered why it is refreshing the backoffs; I was suspicious that this method should be deleted.